### ISSUE TITLE
Segmentation Fault in findmap()

### ISSUE THREAD

**Issue by weekswp** [spatala/ppm3d#5]
Hi there! Trying to use this package and am running into issues with the findmap() function. The point_match library appears to compile effectively but as soon as findmap() calls the point_match() function, the code experiences a Segmentation Fault (core dumped). I was wondering if this was a common issue or if anyone could help me with this. Thanks!

**Comment by spatala** [comment]
Hello,
Not necessarily a common issue. I noticed that you closed the issue. Could you provide us more details for future reference. Thanks.

**Comment by weekswp** [comment]
Thanks for responding! I was using the motifextraction implementation of the ppm3d library at https://github.com/paul-voyles/motifextraction and I realized after posting this issue that the ppm3d.py code in this implementation wasn't feeding the right number of arguments into the point_match call within the find_map function.

Presumably, this is why the code was causing a Segmentation Fault and when I altered the code to reflect the function from this ppm3d repository, it worked. I hope this explains the issue I was having.
